### DISCUSSION TITLE
Error: Signature verification failed

### DISCUSSION THREAD

**Discussion by masterbrews** [phantom/docs#86] [Help & Support] [answered]
### Summary

Since the multichain update some months ago, our Solana staking / unstaking program generates "Error: Signature verification failed" when users attempt to use it with Phantom.

Previous versions of Phantom work fine, as do other Solana-enabled wallets.

We had a case open with Professor in Discord, and even sent him one of our NFTs to debug the issue (many more details were shared with him there); he eventually stopped replying for weeks and we gave up - but still need this fixed for our community.

Thanks,

### Example

https://heroesandvillains.io/stake

### Steps to Reproduce

You would need us to send you one of our NFTs (as we did Professor...)

### Phantom Version

23.11.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hi @masterbrews,

Thank you for writing in!

My apologies that we have not recently followed up! I will check with @0xproflupin and determine where we are at with this issue.

**Reply by masterbrews** [reply]
Great, much thanks.

**Reply by adamdelphantom** [reply, marked as answer]
Hi @masterbrews,

I checked with @0xproflupin and the best guidance he can provide is as he originally stated:

> So this happens if you’re trying to use signTx and not reassigning the transaction explicitly 
> So in your code I see you’re using anchor for implicit signing. I would recommend building the tx but not using anchor for signing it
